test/evaluation/submissionRequirements.spec.ts

```diff
@@ -13,180 +13,176 @@ const HOLDER_DID = 'did:example:ebfeb1f712ebc6f1c276e12ec21';
-    expect(presentationSubmission.definition_id).toEqual("32f54163-7166-48f1-93d8-ff217bdb0653");
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
+    expect(() => evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential)).toThrowError('Count: expected: 1 actual: 2 at level: 0');
   });
 
   it('Evaluate submission requirements all from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
     pdSchema.submission_requirements = [pdSchema.submission_requirements[7]];
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     vpSimple.holder = HOLDER_DID;
-    evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
     expect(() => evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential)).toThrowError('Not all input descriptors are members of group B');
   });
 
   it('Evaluate submission requirements all from group A and 2 from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
     pdSchema.submission_requirements = [pdSchema.submission_requirements[8]];
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     vpSimple.holder = HOLDER_DID;
-    evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
     const result: PresentationSubmission = evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential);
     expect(result).toEqual(expect.objectContaining({
       definition_id: '32f54163-7166-48f1-93d8-ff217bdb0653',
       descriptor_map: [
         { 'format': 'ldp_vc', 'id': 'Educational transcripts', 'path': '$.verifiableCredential[0]' },
-        { 'format': 'ldp_vc', 'id': 'Educational transcripts 1', 'path': '$.verifiableCredential[1]' }
+        { 'format': 'ldp_vc', 'id': 'Educational transcripts 1', 'path': '$.verifiableCredential[1]' },
+        { 'format': 'ldp_vc', 'id': 'Educational transcripts 2', 'path': '$.verifiableCredential[2]' }
       ]
     }));
   });
 
   it('Evaluate submission requirements min 1: (all from group A or 2 from group B)', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.